payments-core/src/main/java/com/stripe/android/model/ConfirmPaymentIntentParams.kt

```diff
@@ -28,8 +29,7 @@ data class ConfirmPaymentIntentParams internal constructor(
     val sourceParams: SourceParams? = null,
     val sourceId: String? = null,
 
-    @Deprecated("Will be removed in an upcoming major version.")
-    val extraParams: Map<String, Any>? = null,
+    private val _extraParams: ExtraParams = ExtraParams(),
```

Comment:
should we kept using  `extraParams` here? this seems not following naming conventions for other properties